tox.ini

```
@@ -0,0 +1,11 @@
+[tox]
+envlist =
+    py37,
+    py37-flake8
+
+[testenv]
+commands = python ./runtests.py
+
+[testenv:flake8]
```

I believe this name needs to be changed:
```suggestion
[testenv:py37-flake8]
```